sohamb117.md

```diff
@@ -0,0 +1,52 @@
+---
+name: "@sohamb117"
+project: "Custom Locus Keyboard"
+---
+
+# Modified Locus Keyboard
+
+## Summary
+
+I'm going to build a split 45% keyboard using a stack of cut acrylic parts. The goal is to build a completely different type of keyboard from the one I am using right now, and to practice improve my skills at soldering, 3D printing, silicone casting, and modification. I'm excited to work on a completely different (and far more in-depth) project than most of my other hardware work, as this will require not just computer science skills but also a knowledge of materials and manufacturing.
+
+## Plan
+
+1) Go to Home Depot and compare different plastics based on their flexibility and resonance (this is just to decide what material to use)
+2) Submit an order for the laser-cut case
+3) 3D print tracing plates for the foam inserts, as well as a mold for the silicone
+4) Cut the foam using the 3d-printed plates
+5) Pour the silicone into the mold, let it harden, and then remove it
+6) Modify PCB to add flex cuts, using a 3d-printed map and a Dremel (this is risky, so I might not do it, depending on how flexible the PCB is in-hand)
+7) Tune stabilizers, using 3d-printed inserts to reduce opening tolerances and lubricants to prevent scratching
+8) Add MillMax 3305 sockets to PCB
+9) 3D-print switch films
+10) Install all the custom foam and silicone
+11) Install stabilizers
+12) Lube and film switches
+13) Install switches
+14) Complete assembly of the case
+15) Configure Vial development environment
+16) Create firmware through Vial
+
+Total Expected Time: 15 Hours (plus cooling and printing time)
+
+## Budget
+
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+| Case and PCB | [P3DStore](https://p3dstore.com/products/locus-invisibolt-acrylic-gasket-mount-case-and-pcb) | $135   |
+| 5x KTT HaluHalo Switches | [DangKeebs](https://dangkeebs.com/products/ktt-haluhalo?variant=39959357194412)  | $15.00 |
+| Krytox 205g0 | [DangKeebs](https://dangkeebs.com/products/205g0?variant=36556464783521)  | $12.00 |
+| MillMax 0305 | [Mouser](https://www.mouser.com/ProductDetail/Mill-Max/0305-2-15-80-47-80-10-0?qs=QtQX4uD3c2Uys0ai6Tr8NQ%3D%3D)  | $17.89  |
+| Stabilizers | [Thockpop](https://thockpop.com/product/aeboards-staebies-v2/) | $22.00 |
+| Keycaps | [AliExpress](https://www.aliexpress.us/item/3256804603576206.html) | $26.39 |
+| Total           |                                       | $241.28 |
+
+(I've already purchased foams as well as silicone and masking tape)
+
+
+## Microphone project (working with @KaiDevrim)
+
+|Product     | Supplier/Link       | Cost |
+| --------------- | ------------------------------------- | ------ |
+|SSM2019 | [Mouser](https://www.mouser.com/ProductDetail/Analog-Devices/SSM2019BNZ?qs=WIvQP4zGanhHwIaXxt763A%3D%3D) | $6.41 |
```

Comment:
As discussed on Slack, could you add this to Kai's proposal as it is my understanding that it relates to his project? The money for this would go into Kai's HCB account, and you would have to be added to it.